ForkMe-Mobile/app/src/main/java/dlei/forkme/gui/activities/github/UserRepositoriesViewActivity.java

```diff
@@ -0,0 +1,291 @@
+        Log.d("UserReposViewActivity: ", "created");
+        mProgressBarSpinner = (ProgressBar)findViewById(R.id.progress_bar_spinner);
+
+        Intent i = getIntent();
+        String userLogin = i.getStringExtra("userLogin");
+
+        // Set up components of RecyclerView.
+        mRecyclerViewRepositories = (RecyclerView) findViewById(R.id.repositoryRecyclerView);
+        mAdapterRepositories = new RepositoryRecyclerViewAdapter(mRepositories);
+        mLayoutManager = new LinearLayoutManager(this);
+
+        // Set up RecyclerView.
+        mRecyclerViewRepositories.setLayoutManager(mLayoutManager);
+        mRecyclerViewRepositories.setItemAnimator(new DefaultItemAnimator());
+        mRecyclerViewRepositories.setAdapter(mAdapterRepositories);
+
+        // Set up lines between items in list.
+        DividerItemDecoration dividerItemDecoration = new DividerItemDecoration(mRecyclerViewRepositories.getContext(),
+                mLayoutManager.getOrientation());
+        mRecyclerViewRepositories.addItemDecoration(dividerItemDecoration);
+
+        // Set up UI elements.
+        mAvatarIconImage = (AppCompatImageView) findViewById(R.id.userAvatarIconImage);
+        mUserNameText = (AppCompatTextView) findViewById(R.id.userNameLoginText);
+        mUserBioText = (AppCompatTextView) findViewById(R.id.userBioText);
+        mUserFollowerCountText = (AppCompatTextView) findViewById(R.id.followersCountText);
+        mUserFollowingCountText = (AppCompatTextView) findViewById(R.id.followingCountText);
+        mUserRepositoryCountText = (AppCompatTextView) findViewById(R.id.repositoryCountText);
+        mUserStarIconImage = (AppCompatImageView) findViewById(R.id.userStarredIconImage);
+        mDivider = findViewById(R.id.viewHeaderSeparator);
+        mRepositoriyIconImage = (AppCompatImageView) findViewById(R.id.userRepositoryIconImage);
+        mFollowersText = (AppCompatTextView) findViewById(R.id.userFollowerText);
+        mFollowingText = (AppCompatTextView) findViewById(R.id.userFollowingText);
+
+
+        mHeaderViews.add(mAvatarIconImage);
+        mHeaderViews.add(mUserNameText);
+        mHeaderViews.add(mUserBioText);
+        mHeaderViews.add(mUserFollowerCountText);
+        mHeaderViews.add(mUserFollowingCountText);
+        mHeaderViews.add(mUserRepositoryCountText);
+        mHeaderViews.add(mUserStarIconImage);
+        mHeaderViews.add(mDivider);
+        mHeaderViews.add(mRepositoriyIconImage);
+        mHeaderViews.add(mFollowersText);
+        mHeaderViews.add(mFollowingText);
+        this.getGithubUser(userLogin);
+        toogleViews(mHeaderViews, View.GONE);
+    }
+
+    /**
+     * Used to hide and unhide views when waiting for HTTP calls.
+     * @param views List of views to change visibility of.
+     * @param visability to set for ech view in views.
+     */
+    private void toogleViews(List<View> views, int visability) {
```

Comment:
Alternatively you could set the ViewGroup's visibility instead. Also, why not just .setVisibility() directly? You have to have a line for each time you add a view to the 'views' list anyway.